webapp/channels/src/actions/post_actions.ts

```diff
@@ -323,9 +324,16 @@ export function unpinPost(postId: string): ActionFuncAsync<boolean> {
 }
 
 export function setEditingPost(postId = '', refocusId = '', isRHS = false): ActionFunc<boolean, GlobalState> {
+    const getFilesForPost = makeGetFilesForPost();
+
     return (dispatch, getState) => {
         const state = getState();
-        const post = PostSelectors.getPost(state, postId);
+        let post = PostSelectors.getPost(state, postId);
+
+        if (post.file_ids?.length) {
```

Comment:
lets add a comment here for clarification

Reply:
Done.